continuous-integration/travis/install.sh

```diff
@@ -0,0 +1,9 @@
+MINICONDA_URL="http://repo.continuum.io/miniconda"
+MINICONDA_FILE="Miniconda-3.5.5-Linux-x86_64.sh"
```

Comment:
I proposed this change for the original project too https://github.com/rmcgibbo/python-appveyor-conda-example/pull/2